src/Kdyby/Events/EventManager.php

```diff
@@ -171,7 +171,7 @@ public function hasListeners($eventName)
 	 * Adds an event listener that listens on the specified events.
 	 *
 	 * @param string|array $events The event(s) to listen on.
-	 * @param Doctrine\Common\EventSubscriber|array $subscriber The listener object.
+	 * @param Doctrine\Common\EventSubscriber|array|callable $subscriber The listener object.
```

Comment:
As @vojtech-dobes said somewhere, Doctrine\Common\EventSubscriber does not exist.

The dependency is missing?